config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -134,7 +134,7 @@ periodics:
       - name: INIT_WITH_PROVIDERS_CONTRACT
         value: v1beta1
       - name: INIT_WITH_KUBERNETES_VERSION
-        value: v1.24.0
+        value: v1.25.0
```

Comment:
Let's do this separately after we changed the corresponding stuff in core CAPI.

It's just easier to test first on a PR against CAPI